src/multiplayer-servers/multiplayer-services/steelshield/steelshield-for-agones.md

```diff
@@ -0,0 +1,68 @@
+# SteelShield for Agones
+
+SteelShield for Agones protects the Agones game server traffic from your GameFabric installation
```

Comment:
A Question, do we want to refer to it as `Agones` game server traffic. Why not the GameFabric game server traffic?

Reply:
Yeah I also noticed that this phrase sounds weird with the new section naming. Can you have another look at the introduction sentence?